syft/serde.py

```diff
@@ -496,111 +499,60 @@ def _detail_ellipsis(ellipsis: bytes) -> Ellipsis:
     return ...
 
 
-# PointerTensor
-
-
-def _simplify_pointer_tensor(ptr: PointerTensor) -> Dict:
+def _simplify_pointer_tensor(ptr: PointerTensor) -> tuple:
     """
     This function takes the attributes of a PointerTensor and saves them in a dictionary
-
     Args:
         PointerTensor: a PointerTensor
```

Comment:
Should be ptr (PointerTensor) as above. Or just ptr because Napoleon can handle type annotations